x-pack/plugins/event_log/scripts/mappings.js

```diff
@@ -189,6 +189,18 @@ exports.EcsCustomPropertyMappings = {
       },
     },
   },
+  action: {
+    properties: {
+      name: {
+        ignore_above: 1024,
+        type: 'keyword',
+      },
+      uuid: {
```

Comment:
haven't gotten to the point where this is used, so I'm curious what it's a UUID *for*!  execution uuid?